### DISCUSSION TITLE
Missing row rank method

### DISCUSSION THREAD

**Discussion by ORESoftware** [josdejong/mathjs#2857] [Ideas]
https://mathjs.org/docs/datatypes/matrices.html

I do not see a method for `column rank` or `row rank`, this is perplexing.

**Comment by josdejong** [comment]
There is indeed no function `rank` available right now. Help implementing this would be welcome.

**Reply by ORESoftware** [reply]
lets make this happen! I can help

**Reply by josdejong** [reply]
Thanks for your offer to pick this up Alexander! Please let me know if you need help getting up to speed.
